test/test_cuda.py

```diff
@@ -2456,6 +2456,14 @@ def test(x, ia, ib):
     def test_advancedindex_big(self):
         _TestTorchMixin._test_advancedindex_big(self, lambda t: t.cuda())
 
+    @unittest.skipIf(not TEST_LARGE_TENSOR, "not enough memory")
```

Comment:
should this be a slowTest?  How long does it take?

Reply:
Yes, it takes ~13 seconds